docs/_posts/2022-02-21-release-1-0-0.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Python-TUF reaches version 1.0.0"
+author: Jussi Kukkonen and Lukas Pühringer
+---
+
+The Python-TUF community is proud to announce the release of Python-TUF 1.0.0.
+The release, which is available on [PyPI](https://pypi.org/project/tuf/) and
+[GitHub](https://github.com/theupdateframework/python-tuf/), introduces new
+stable and more ergonomic APIs.
+
+<img align="right" src="../../../tuf-icon-color.png" width="200"/>
```

Comment:
Isn't this
```suggestion
<img align="right" src="../tuf-icon-color.png" width="200"/>
```
?

Also  https://github.com/theupdateframework/python-tuf/blob/develop/docs/tuf-icon-color.png is in favicon resolution, so I guess that's a bit small.

Reply:
> oops, favicon was not the intention.

We could copy [the image I used for the ssl blog](https://github.com/lukpueh/ssl-site/blob/bb0b59634c2807c52d926bce3e0783c400d56664/img/blog/tuf-icon.png), it has an appropriate resolution. FYI: The original is taken from [cncf/artwork#tuf-icon-color.png](https://github.com/cncf/artwork/blob/master/projects/tuf/icon/color/tuf-icon-color.png).
> 
> The path I used _should_ work, although it might not be the best way to handle it: The issue with asset paths is that they are for the final URL (jekyll does not mangle them) so we either need to use absolute paths, which means `/python-tuf/FILENAME` or we use relative paths which is what I did: the post path is `/python-tuf/2022/02/21/POST-TITLE.html` so we need to go up three levels to `../../../FILENAME`.
> 
> There may be a jekyll config that allows defining a the asset lookup path but I've not found one yet

Thanks for the explanation!